shopify/resources/fulfillment_event.py

```diff
@@ -0,0 +1,21 @@
+from ..base import ShopifyResource
+
+class FulfillmentEvent(ShopifyResource):
+    _prefix_source = "/orders/$order_id/fulfillments/$fulfillment_id/"
+    _singular = 'event'
+    _plural = 'events'
+
+    @classmethod
+    def _prefix(cls, options={}):
+        order_id = options.get("order_id")
+        fulfillment_id = options.get('fulfillment_id')
+        event_id = options.get("event_id")
+
+        return "%s/orders/%s/fulfillments/%s" % (
+            cls.site, order_id, fulfillment_id)
+
+    def save(self):
+      status = self.attributes['status']
+      if status not in ['label_printed', 'label_purchased', 'attempted_delivery', 'ready_for_pickup', 'picked_up', 'confirmed', 'in_transit', 'out_for_delivery', 'delivered', 'failure']:
```

Comment:
Unsure if hardcoding a list of accepted status' is the best way to go about this

Reply:
As for whether to hard code them: Are these likely to change in the future at all/frequently? And is there another way to reference them?

Reply:
No other way to reference status as far as I know

Reply:
Yeah I didn't think so, but wanted to ask. I would maybe just assign a variable for the array but otherwise if that's the only option, that's the only option 🤷